<!-- -->
<launch>

	<node name="esr_radar" pkg="esr_radar" type ="esr_radar" output="screen">
		<param name ="is_pubBoundingBox" type ="bool" value="True"/>
		<param name ="is_sendMsgToEsr"  type ="bool"  value="True"/>
		<param name ="in_port_name" value="/dev/U2"/>
		<param name ="out_port_name" value="/dev/U3"/>
	</node>
</launch>
